dbcrossbar/fixtures/more_pg_types.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE more_pg_types (
+    c1 character,
+    c2 character(2),
+    c3 character (3)
```

Comment:
@emk Is the space here between `character` and `(3)` a deliberate syntax variation test?

Reply:
@wboykinm Yeah, I like to mix things up a bit in the test cases sometimes. No point in testing the same thing twice. :-)